docs/appendices/release-notes/5.10.1.rst

```diff
@@ -112,3 +112,7 @@ Fixes
   :ref:`numeric <type-numeric>` values. Previously, the values where implicitly
   casted to ``STRING`` and the aggregations where using alphanumeric ordering,
   instead of numeric ordering.
+
+- Fixed an issue that led to ``ArithmeticException`` when using
+  :ref:`AVG <aggregation-avg>` with ``NUMERIC`` type and result was an infinite
```

Comment:
maybe user the more formal term: `irrational number`, just a suggestion.

Reply:
doesn't have to be irrational, 1/3 is rational (because we can represent it as integer nominator and denominator) it's just happens to be infinite.

Reply:
true, sorry.